ui/src/app/pages/workspace/workspace-share.spec.tsx

```diff
@@ -191,15 +191,12 @@ describe('WorkspaceShare', () => {
     const wrapper = component();
     await waitOneTickAndUpdate(wrapper);
 
-    expect(
-      wrapper
-        .find('[data-test-id="harry.potter@hogwarts.edu-user-role"]')
-        .first()
-        // Property 'isDisabled' does not exist on type 'HTMLAttributes'. Did you mean 'disabled'?
-        // TODO RW-5572 confirm proper behavior and fix
-        // eslint-disable-next-line @typescript-eslint/dot-notation
-        .props()['isDisabled']
-    ).toBe(true);
+    const roleSelectProps = wrapper
+      .find('[data-test-id="harry.potter@hogwarts.edu-user-role"]')
+      .first()
+      .props() as object;
+    expect(roleSelectProps.hasOwnProperty('isDisabled')).toBe(true);
```

Comment:
nit: I don't think it's necessary to make this distinction. L199 will already handle this case and provide an equally informative error message

Reply:
thanks.